che-incubator/che-code
# Use universal developer image for dev container

### What does this PR do?
- Inherits che-code-dev image from Universal Developer Image
- Adds few commands to build che-code image with podman

### What issues does this PR fix?

https://github.com/eclipse/che/issues/22292

### How to test this PR?
Do following:
- use the dogfooding instance to create a workspace from the current branch https://github.com/che-incubator/che-code/tree/switch-to-udi-merge-test
- use devfile commands to compile and run VS Code with `yarn`
- use devfile commands to compile and then build `che-code` image with `podman`

![Screenshot from 2023-09-07 16-01-35](https://github.com/che-incubator/che-code/assets/1655894/70ae061f-b75c-4419-ace6-d85939e973aa)

Should have like following:

![Screenshot from 2023-06-26 17-50-43](https://github.com/che-incubator/che-code/assets/1655894/3ecc8457-7958-4ad1-b11d-028c80558320)

![Screenshot from 2023-06-26 17-49-35](https://github.com/che-incubator/che-code/assets/1655894/2c855446-9749-46a4-8108-7579d678d253)

---

### Note that:

Depends on https://github.com/devfile/developer-images/pull/117
